components/input/navigationRectangle.js

```diff
@@ -1,58 +1,103 @@
-import { StyleSheet, View } from "react-native";
-import { Icon, Text } from "react-native-paper";
+import { View } from "react-native";
+import { Divider, Icon, Text } from "react-native-paper";
 
 import { getIconByKey } from "~/Utility";
 
-export default function NavigationRectangle({
-  drillInfo,
-  attemptShots,
-  inputValues,
-  shotIndex,
-}) {
+export default function NavigationRectangle({ drillInfo, inputValues, shot }) {
+  console.log("inputValues", inputValues);
+  const keys = Object.keys(inputValues);
   return (
-    <View style={styles.container}>
-      <View style={styles.rowContainer}>
-        <Text>
-          Shot {shotIndex}/{attemptShots.length}
-        </Text>
-        <Text>
-          Target: {attemptShots[shotIndex - 1].target}
-          {drillInfo.requirements[0].distanceMeasure}
+    <View
+      style={{
+        backgroundColor: "#d9d9d9",
+        padding: 10,
+        paddingLeft: 20,
+        paddingRight: 20,
+        borderRadius: 10,
+        maxHeight: 250,
+        width: "80%",
+        overflow: "hidden",
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "space-between",
+        elevation: 5,
```

Comment:
<img width="1014" alt="Screenshot 2024-04-23 at 1 41 28 PM" src="https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/41023671/6f4980ec-5342-464a-bd58-bc74ad75f464">
I think you had some drop shadow here before, iOS doesn't seem to have the elevation thingie

Reply:
I was able to add back the shadow for IOS and have it very similar between IOS and Andriod